internal/app/runc_memfd_bind/runc_memfd_bind.go

```diff
@@ -0,0 +1,135 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
should we also copy over the license text from source?

Reply:
if we copied a significant amount of code, let's add the original license and link in the commands